### DISCUSSION TITLE
[RELEASE] Aperture

### DISCUSSION THREAD

**Discussion by acronmd** [nzp-team/nzportable#770] [Map Releases]
This is my first attempt at making a map! Or mapping anything really, I don't know of any bugs and I've tested on a New 3DS, Vita, and PC and have gotten good performance (although visually, it's designed with NX/VITA and PC in mind because of transparent glass and brighter lighting)
I hope everyone likes it!

![Preview1](https://github.com/nzp-team/nzportable/assets/78101120/980924bf-6a76-407d-b26c-d5cab60da093)
![Preview2](https://github.com/nzp-team/nzportable/assets/78101120/0e280584-2855-4e9a-bada-77808b8ed20a)
![Preview3](https://github.com/nzp-team/nzportable/assets/78101120/8ec466e9-2a2c-43c4-8c0a-8efee325714e)
![Preview4](https://github.com/nzp-team/nzportable/assets/78101120/b2a8c064-87b2-44d3-bced-7b533c36b030)

It features 5 perk machines, spare change, pack-a-punch, and a few wall buys.

Download is hosted on the Internet Archive and can be found below 
# [DOWNLOAD](https://archive.org/download/aperture_20240608/Aperture.zip)

Thank you to everyone in the discord for being very helpful with my questions.

**Comment by PHDFLOPPERGUY56** [comment]
Trying this map asap this looks sick!!! Just from reading the description I would love to see all 8 perk machines that nzp has so far on the map but at least it looks like theres PhD :)

**Reply by acronmd** [reply]
The 5th perk machine is a random one! The map was rather small and I didn't want to clutter up any specific areas, so I felt that it was a good compromise. I also made sure there were a few good places to use PhD!

**Reply by PHDFLOPPERGUY56** [reply]
How random is it? If the random perk is the one in the pap room it seems like Ive been getting stamin up every game, would be awesome to see that location swap with another perk machine not currently in the map every round (ex. reaper collection mod bo3) to be able to obtain all the perks
Edit : finally got double tap in that spot, i think in order for the perk to change you have to exit to the main menu and go back in, guess you cant just restart the map to do it

**Comment by bigjackson** [comment]
gonna try out this map probably later, but i saw some of the screenshots you sent on the discord and the map looked fire 🙏

**Comment by MotoLegacy** [comment]
Okay, wow. Lol.
I've always loved these themed maps in the Call of Duty scene, and love when they come to NZ:P. They're like junk food. But this is very well crafted, and very pretty on top. All it's missing is a music easter egg, Still Alive, maybe? ;)

**Comment by ZakiGriiez** [comment]


**Reply by acronmd** [reply]
I used GCFScape and VTFEdit on my copy of the original game to extract whatever I needed, and resized them / made small edits in Clip Studio

**Comment by DeepBlueKeaton** [comment]
It's a very fun map, but it crashes my game every Wave 5. I'm on the New 3DS version BTW.
